crates/claims/crates/vc/src/syntax/context.rs

```diff
@@ -65,113 +126,77 @@ impl<V, T> Borrow<ssi_json_ld::syntax::Context> for Context<V, T> {
     }
 }
 
-impl<'de, V: RequiredContext, T: RequiredContextList> Deserialize<'de> for Context<V, T> {
-    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-    where
-        D: serde::Deserializer<'de>,
-    {
-        struct Visitor<V, E>(PhantomData<(V, E)>);
+impl<V, T> From<Context<V, T>> for ssi_json_ld::syntax::Context {
```

Comment:
Did you want to have a `TryInto` as well, or a `::new` function?

Reply:
Do you mean this?
https://github.com/spruceid/ssi/blob/4a109f169dd5d73c7cd889deaebac6f94506d824/crates/claims/crates/vc/src/syntax/context.rs#L146

Reply:
oh I didn't know that existed, I could have made my thing work then (but this is nicer to use!)